### PR TITLE
Added missing event

### PR DESCRIPTION
**Pull request by shyim**

### CODE REVIEW

**Comment by dnoegel** [comment]
Thanks @shyim. IIRC we intentionally did not document that one, as it is not compatible with multiple plugins (only one plugin "wins"). I am not saying we should not document it - but that was the intention behind it back then. What do the others think? Should we just add it for a complete overview?

**Comment by Argee88** [comment]
I really think all of the available events (and possibly hooks as well) should be documented. It is just taking so much time to even figure out the names. If you managed to find the event you still have to find out what arguments/data you can retrieve from them... I'm pretty sure that I could speed up my development by some 20% if events were documented better.

**Comment by mitelg** [comment]
as @dnoegel said, this event is quite dangerous, but I also agree with @Argee88 we should document all events. 
@shyim maybe you can add a hint or something like that below the event list, regarding the problem @dnoegel mentioned

**Comment by t2oh4e** [comment]
@shyim Something like that: "Using this event is not recommended as it is not compatible with multiple plugins."

File: source/developers-guide/controller/index.md [hunk 1]
```diff
@@ -167,6 +167,7 @@ The order of the events is the following:
 * Enlight_Controller_Action_PreDispatch
 * Enlight_Controller_Action_PreDispatch_Frontend
 * Enlight_Controller_Action_PreDispatch_Frontend_Index
+* Enlight_Controller_Action_Frontend_Index
```
Comment (mitelg) [inline source/developers-guide/controller/index.md]: some hint would be nice. see comment

**Comment by mitelg** [comment]
@shyim any update here?

**Comment by mitelg** [comment]
closing due to inactivity

**Pull request closed without merging**